sources/Launcher/DYNMarginCalculationLauncher.cpp

```diff
@@ -535,15 +547,14 @@ MarginCalculationLauncher::findOrLaunchLoadIncrease(const boost::shared_ptr<Load
 void
 MarginCalculationLauncher::launchLoadIncrease(const boost::shared_ptr<LoadIncrease>& loadIncrease,
     const double variation, SimulationResult& result) {
-  if (nbThreads_ == 1)
-    std::cout << "Launch loadIncrease of " << variation << "%" <<std::endl;
+  std::stringstream ss;
```

Comment:
Please keep the condition on number of threads

Reply:
OK